pkg/promql/test.go

```diff
@@ -568,10 +568,11 @@ func (t *Test) clear() {
 	t.storage = NewTestStorage(t)
 
 	opts := EngineOpts{
-		Logger:     nil,
-		Reg:        nil,
-		MaxSamples: 10000,
-		Timeout:    100 * time.Second,
+		Logger:                   nil,
+		Reg:                      nil,
+		MaxSamples:               10000,
+		Timeout:                  100 * time.Second,
+		NoStepSubqueryIntervalFn: func(int64) int64 { return durationMilliseconds(1 * time.Minute) },
```

Comment:
We should maybe add an feature-request issue to make this configurable

Reply:
Ahh, can you explain this bit more? Is this for upstream?